source/val/validate_extensions.cpp

```diff
@@ -1082,6 +1082,23 @@ spv_result_t ValidateExtInstImport(ValidationState_t& _,
   return SPV_SUCCESS;
 }
 
+spv_result_t ValidateExtInstWithForwardRefs(ValidationState_t& _,
+                                            const Instruction* inst) {
+  if (!_.HasExtension(kSPV_KHR_relaxed_extended_instruction)) {
```

Comment:
This should be handled by the parser already. Since the grammar has no core version and a single extension listed.

Reply:
Right, removed this code, and was done,thansk!